packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -68,23 +68,22 @@ export class HouseController {
     description: 'house retrieved successfully',
     type: HouseResponseDto,
   })
-  @ApiNotFoundResponse({ description: 'user does not have a house' })
-  async getHouse(@User() user: DecodedIdToken): Promise<HouseResponseDto> {
+  async getHouse(
+    @User() user: DecodedIdToken,
+  ): Promise<HouseResponseDto | null> {
     const userDoc = await this.userStoreService.findOneByFirebaseId(user.uid);
     const house = await this.houseStoreService.findOne(userDoc.house);
-    if (house)
+    if (house) {
       return {
         code: house.code,
         address: house.address,
         email: house.email,
         owner: user.uid,
         name: house.name,
       };
-    else
-      throw new HttpException(
```

Comment:
The user not having a house is expected, therefore I prefer returning null over a 404.